app/com/linkedin/drelephant/spark/legacydata/SparkEnvironmentData.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2016 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.linkedin.drelephant.spark.legacydata;
+
+import java.util.Properties;
+
+
+/**
+ * This data class holds Spark environment data (Spark properties, JVM properties and etc.)
+ */
+public class SparkEnvironmentData {
+  private final Properties _sparkProperties;
```

Comment:
That's because HadoopApplicationData interface expects it to be Properties. Probably a legacy of what mapreduce's getconf returns. Now, it will be a big change to change it. I am going to leave it as it is for now.